src/sentry/models/servicehook.py

```diff
@@ -43,7 +43,7 @@ def generate_secret():
     return uuid4().hex + uuid4().hex
 
 
-@region_silo_only_model
+@control_silo_only_model
```

Comment:
I don't think this can become a Control Silo model. Service hooks are fired from post_process_group which is a high-throughput task related to ingestion. They also have several endpoints related to projects.

We might need to approach this from the other side and break the foreign key to `ApiApplication`.

Reply:
Hmm.  Ok, let me rexamine.